points.py

```diff
@@ -1,35 +1,40 @@
 import sys
 import requests
 import json
+import operator

```

Comment:
what does this library allow us to do?

Reply:
See the operator functions in the change_points